Alerting: Add LogicOr operation option (#89258)
grafana/grafana: docs/sources/alerting/fundamentals/alert-rule-evaluation/_index.md

You can also set the pending period to zero to skip it and have the alert fire immediately once the condition is met.

## Condition operator

> There are several condition operators available:
> 
> - **and**: Two conditions before and after it must be true for the overall condition to be true.
> - **or**: If one of conditions before and after it is true, the overall condition is true.
> - **logic-or**: If the condition before logic-or is true, the overall condition is immediately true, without evaluating subsequent conditions.
> 
> Here are some example of operators:
> 
> - `TRUE and TRUE or FALSE and FALSE` will evaluate to `FALSE` because last two conditions return `FALSE`.
> - `TRUE and TRUE logic-or FALSE and FALSE` will evaluate to `TRUE` because preceding condition return `TRUE`.
> 

Reviewer: ```suggestion
There are several condition operators available.

- **and**: Two conditions before and after must be true for the overall condition to be true.
- **or**: If one of conditions before and after are true, the overall condition is true.
- **logic-or**: If the condition before logic-or is true, the overall condition is immediately true, without evaluating subsequent conditions.

Here are some examples of operators.

- `TRUE and TRUE or FALSE and FALSE` evaluate to `FALSE`, because last two conditions return `FALSE`.
- `TRUE and TRUE logic-or FALSE and FALSE` evaluate to `TRUE`, because the preceding condition returns `TRUE`.

```
